scripts/go/configs/.golangci.toml

```diff
@@ -11,6 +11,13 @@ min-occurrences = 5
 [linters-settings.exhaustive]
 default-signifies-exhaustive = true
 
+[linters-settings.revive]
+ignore-generated-header = false
+severity = "warning"
+confidence = 3
+errorCode = 0
+warningCode = 0
```

Comment:
Is it necessary to make `revive` not fail with non-zero exit code? Don't the other settings stop revive from complaining?

Reply:
It's not - I used the default revive configuration [as shown here](https://github.com/mgechev/revive#recommended-configuration) since we are going to work on them anyway. Can change if you like

Reply:
AFAICT, that is _not_ the default configuration. You shouldn't add configuration parameters that don't override the defaults anyway, right?

Reply:
Based on [this comment](https://github.com/mgechev/revive/issues/241#issuecomment-543291518) I think I'm right in thinking that this will make `revive` exit with non-zero code on errors and warnings. That will mean failures from `revive` will be ignored.